observability/tools/loki.md

```diff
@@ -0,0 +1,39 @@
+# Loki
+
+Loki is a horizontally-scalable, highly-available, multi-tenant log aggregation system, created by Grafana
+Labs inspired by the learnings from Prometheus. Loki is commonly referred as Prometheus but for logs, which
+makes total sense once both tools follow the same architecture, which is a agent collecting metrics in each
+of the components of the software system, a server which stores the logs and also the Grafana dashboard, which
+access the loki server to build its visualizations and queries. That being said, Loki has three main
+components:
+
+## Promtail
+
+Its the agent portion of Loki. It can be used to grab metrics from several different places, like var/log/ for
```

Comment:
Nit: promtail deals with log file vs metric sources (like Cortex/Prometheus does). I would replace the term `metrics` here with `logs`

There is a unified agent from Grafana which deals with both log files & metrics: https://grafana.com/docs/grafana-cloud/agent/

Reply:
That makes total sense as well!